src/wp-includes/block-patterns.php

```diff
@@ -207,6 +230,7 @@ function _load_remote_block_patterns( $deprecated = null ) {
  * Register `Featured` (category) patterns from wordpress.org/patterns.
  *
  * @since 5.9.0
+ * @since 6.2.0 Normalize the pattern from the API (snake_case) to the format expected by `register_block_pattern` (camelCase).
```

Comment:
```suggestion
 * @since 6.2.0 Normalized the pattern from the API (snake_case) to the
 *                        format expected by `register_block_pattern()` (camelCase).
```